src/server/page.tsx

```diff
@@ -164,53 +78,67 @@ const styles = (format: Format): string => `
     }
 `;
 
-function page(
-    imageSalt: string,
-    renderingRequest: RenderingRequest,
-    getAssetLocation: (assetName: string) => string,
-): Page {
-    const item = fromCapi({ docParser, salt: imageSalt })(renderingRequest);
-    const shouldHideAds = renderingRequest.content.fields?.shouldHideAdverts ?? false;
-    const hasTweets = includesTweets(renderingRequest.content);
-    const clientScript = map(getAssetLocation)(scriptName(item));
-    const { html: body, css, ids } = compose(extractCritical, renderToString)(
+function renderHead(
```

Comment:
Maybe another one to abstract into a `head` file?

Reply:
I actually did that at one point and then undid it again 🙈